src/lib.rs

```diff
@@ -561,7 +561,7 @@ pub async fn send_assets(
     info!("Creating transfer PSBT...");
 
     #[cfg(not(target_arch = "wasm32"))]
-    let (consignment, psbt, disclosure) = transfer_assets(
+    let (consignment, psbt, disclosure, _change, _previous_utxo, _new_utxo) = transfer_assets(
```

Comment:
Mind if we change these to a `TransferAssetsResponse` struct? Then we can just pass it around.

Reply:
Yeah much better